src/xcode/ENA/ENAUITests/HealthCertificate/CreateHealthCertificate.swift

```diff
@@ -0,0 +1,115 @@
+////
```

Comment:
Could you please replace ALL `tap()`-calls by `tapAndWait()`?
And because `tapAndWait()` has as default timeout `.short`, you can remove that.
Also, because `tapAndWait()` includes `waitForExistence()`, you can remove every `waitForExistence()` in the line before `tapAndWait()`.
All these changes will prevent flaky tests and makes our UITest-code consistent.

Reply:
done